src/multicharge/param.f90

```diff
@@ -32,14 +33,16 @@ subroutine new_eeq2019_model(mol, model)
    !> Electronegativity equilibration model
    type(mchrg_model_type), intent(out) :: model
 
-   real(wp), allocatable :: chi(:), eta(:), kcn(:), rad(:)
+   real(wp), allocatable :: chi(:), eta(:), kcn(:), rad(:), rcov(:)
 
    chi = get_eeq_chi(mol%num)
    eta = get_eeq_eta(mol%num)
    kcn = get_eeq_kcn(mol%num)
    rad = get_eeq_rad(mol%num)
+   rcov = get_covalent_rad(mol%num)
 
-   call new_mchrg_model(model, chi=chi, rad=rad, eta=eta, kcn=kcn)
+   call new_mchrg_model(model, mol, chi=chi, rad=rad, eta=eta, kcn=kcn, &
+      & cutoff=25.0_wp, cn_exp=7.5_wp, rcov=rcov, cn_max=8.0_wp)
```

Comment:
Can you make these values parameters? See [here](https://github.com/tblite/tblite/blob/ccc00f21a41b205e2284fb97bd0417ef5045fc9c/src/tblite/classical/halogen.f90#L56) for example.